deploy/docker-compose/.env

```diff
@@ -0,0 +1,76 @@
+# Container Images
+INFORMATICS_GATEWAY_IMAGE=ghcr.io/project-monai/monai-deploy-informatics-gateway:0.3.0-rc.1
+INFORMATICS_GATEWAY_CONFIG_IMAGE=dwdraju/alpine-curl-jq:latest
+WORKFLOW_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-workflow-manager:develop-nightly-2022-09-12
+TASK_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-task-manager:develop-nightly-2022-09-12
+
+MINIO_IMAGE=minio/minio:latest
+MINIO_MC_IMAGE=minio/mc
+MONGODB_IMAGE=mongo:latest
```

Comment:
For mongo and minio, should we use a specific version? Latest on updates might cause issues